ui/src/hooks/useFileManage.ts

```diff
@@ -238,6 +272,19 @@ const initSocketEvent = (socket: WebSocket) => {
         break;
       }
 
+      case MessageType.PING: {
+        socket.send(JSON.stringify({
+          id: uuid(),
+          type: MessageType.PONG,
+          data: 'pong'
+        }));
+        break;
+      }
+    
+      case MessageType.PONG: {
+        break;
+      }
+
       default: {
         break;
       }
```

Comment:
I'm unable to assess the code because it is cut off with "const fileManageStore = useFileManageStore();". Please ensure that the whole content provided here includes `fileManageStore`, otherwise further analyses will not be valid.

If there's an issue mentioned, please provide more details so I could help you better.